packages/visual-editor/src/components/puck/NearbyLocations.tsx

```diff
@@ -0,0 +1,339 @@
+          { label: "International", value: "international" },
+        ],
+      },
+      phoneNumberLink: {
+        label: "Include Phone Number Hyperlink",
+        type: "radio",
+        options: [
+          { label: "Yes", value: true },
+          { label: "No", value: false },
+        ],
+      },
+    },
+  },
+  coordinate: YextEntityFieldSelector<any, Coordinate>({
+    label: "Coordinates",
+    filter: { types: ["type.coordinate"] },
+  }),
+  radius: {
+    label: "Radius (Miles)",
+    type: "number",
+    min: 0,
+  },
+  limit: {
+    label: "Limit",
+    type: "number",
+    min: 0,
+    max: 50,
+  },
+};
+
+const LocationCard = ({
+  cards,
+  name,
+  address,
+  hours,
+  timezone,
+  mainPhone,
+  backgroundColor,
+}: {
+  cards: NearbyLocationsProps["cards"];
+  name: string;
+  address: any;
+  hours: any;
+  timezone: string;
+  mainPhone: string;
+  backgroundColor?: BackgroundStyle;
+}) => {
+  return (
+    <div className="flex flex-col flew-grow h-full rounded-lg overflow-hidden border">
+      <Section background={backgroundColor} className={`components`}>
+        <Heading level={4}>{name}</Heading>
+        {hours && (
+          <div className="mb-2 font-semibold font-body-fontFamily text-body-fontSize">
+            <HoursStatus
+              hours={hours}
+              timezone={timezone}
+              currentTemplate={
+                cards.hours.showCurrentStatus ? undefined : () => <></>
+              }
+              separatorTemplate={
+                cards.hours.showCurrentStatus ? undefined : () => <></>
+              }
+              timeOptions={{
+                hour12: cards.hours.timeFormat === "12h",
+              }}
+              dayOptions={{
+                weekday: cards.hours.dayOfWeekFormat,
+              }}
+              dayOfWeekTemplate={
+                cards.hours.showDayNames ? undefined : () => <></>
+              }
+              className="h-full"
+            />
+          </div>
+        )}
+        {mainPhone && (
+          <Phone
+            phoneNumber={mainPhone}
+            format={cards.phoneNumberFormat}
+            includeHyperlink={cards.phoneNumberLink}
+          />
+        )}
+        {address && (
+          <div className="font-body-fontFamily font-body-fontWeight text-body-fontSize-sm">
+            <Address address={address} lines={[["line1"]]} />
+          </div>
+        )}
+      </Section>
+    </div>
+  );
+};
+
+const NearbyLocationsComponent: React.FC<NearbyLocationsProps> = (props) => {
+  const {
+    heading,
+    cards,
+    coordinate: coordinateField,
+    radius,
+    limit,
+    styles,
+  } = props;
+  const document = useDocument<any>();
+
+  const coordinate = resolveYextEntityField<Coordinate>(
+    document,
+    coordinateField
+  );
+  const headingText = resolveYextEntityField<string>(document, heading.text);
+
+  const contentEndpoint: string = document?._env?.YEXT_CONTENT_API_KEY;
```

Comment:
This is YEXT_CONTENT_ENDPOINT per your updated CR https://gerrit.yext.com/c/alpha/+/289517/1/gocode/src/yext/publish/visualeditor/server/cmd/visualeditorserver/server/site/createVisualEditorSite.go#58